### ISSUE TITLE
Wrong translation pack

### ISSUE THREAD

**Issue by scommercemage** [magento-l10n/language-en_GB#1]
![image](https://user-images.githubusercontent.com/8902848/83924701-231bda80-a77d-11ea-904f-2ff8186c7a5e.png)

**Comment by git-seb** [comment]
This is now fixed.

**Comment by davidwindell** [comment]
@git-seb nope, the csv is still full of non-english translations like `Daten neuladen`

**Comment by git-seb** [comment]
@davidwindell Hi, thanks for mentioning. I have amended "Reload Data" and "Update Items and Quantities". Expect it to be updated with composer by tomorrow.

**Comment by fballiano** [comment]
@git-seb I think this could be closed
